### DISCUSSION TITLE
Masks and foil textures origin

### DISCUSSION THREAD

**Discussion by Matoseb** [simeydotme/pokemon-cards-css#43] [Q&A] [answered]
Hello,
What a mindblowing project!
I was wondering how you made those image textures (the masks & foils)
Was it all by hand? Or maybe generated with an AI model?
Thanks for your response

**Comment by Matoseb** [comment]
Ah got it! does it come from malie.io ?

**Reply by simeydotme** [reply, marked as answer]
Hi, ummm, yes , the sources are from Malie ... however they were not high enough resolution and contrast... so I did run them through an image upscaler and used imageMagick to process them... the code for that is in the `.sh` file :)

**Reply by simeydotme** [reply]
I can try to find the links for the CDN json files later, I dont know why I didn't put a link in there as an example ... maybe out of respect to the owner (u can join his discord and ask for any info/data https://pkmncards.com/  https://discord.gg/YwGcUba )
